### PR TITLE
Remove deadcode

### PR DESCRIPTION
**Pull request by 100mik**
Removing deadcode detected by `golangci-lint` on [Sonatype Lift](https://lift.sonatype.com/result/longhorn/longhorn-manager/01FHW2JKTWJ3Z4RA5MMT2A53WS?t=CustomTool%20%22golangci-lint%22%7Cdeadcode).

### CODE REVIEW

**Review by shuo-wu — APPROVED** [review]

File: app/driver.go
Comment (shuo-wu) [inline app/driver.go:322]: After rechecking the code, it seems that struct `DaemonSetOps` is not used as well. I think you can remove the struct as well as all methods/functions for it.

File: client/generated_node_input.go
Comment (shuo-wu) [inline client/generated_node_input.go:31]: Similar here. It seems that this whole go file is useless.

File: types/types.go
Comment (100mik) [reply types/types.go:173]: made this change!

File: app/driver.go
Comment (shuo-wu) [inline app/driver.go:318]: Need to remove these lines.

Comment (100mik) [reply app/driver.go]: My bad missed this. Done!

File: types/types.go
Comment (joshimoo) [inline types/types.go:418]: Since the prefix already includes, the `-` sign, the appending of the `-` is not longer needed.

File: types/types.go
Comment (joshimoo) [inline types/types.go:422]: Since the prefix already includes, the - sign, the appending of the - is not longer needed.

File: types/types.go
Comment (joshimoo) [inline types/types.go:174]: NIT: no need for a separate section for this variable.

**Review by joshimoo — CHANGES_REQUESTED** [review]
LGTM, when replacing the usage with the const var, the extra `-` was overlooked. After that is fixed this can be merged.

File: types/types.go
Comment (100mik) [reply types/types.go]: Incorporated suggestions!

**Review by joshimoo — APPROVED** [review]
LGTM

**Pull request merged**
